valhalla/mjolnir/osmaccess.h

```diff
@@ -19,7 +19,7 @@ struct OSMAccess {
   OSMAccess() {
     memset(this, 0, sizeof(OSMAccess));
     osmwayid_ = 0;
-    attributes_ = {0};
+    attributes_ = {.v = 0};
```

Comment:
This is annoying. I had to tweak this syntax to satisfy gcc but now msvc instead complains :crying_cat_face:

Reply:
i think the constructor can be vastly simplified and work on all platforms:

```c++
  OSMAccess() {
    memset(this, 0, sizeof(OSMAccess));
    osmwayid_ = 0;
    attributes_.v = 0;
  }
```